webapp_utils/entrypoint.sh

```diff
@@ -0,0 +1,5 @@
+#!/bin/sh
+
+DIR=$(dirname $0)
+
+exec $DIR/webapp_utils.py
```

Comment:
This produces log message:
~~~
WARNING: This is a development server. Do not use it in a production deployment.
vmaas-webapp-utils
Use a production WSGI server instead.
~~~
You should use production server here (e.g. gunicorn). See vulnerability engine - manager. It's the same case.